Gemfile

```diff
@@ -45,6 +45,8 @@ group :test do
   gem 'webmock', require: false
   gem 'vcr'
   gem 'puffing-billy'
+  gem 'simplecov', require: false
+  gem 'coveralls', require: false
```

Comment:
In order for SimpleCov and Coveralls to run, the `Simplecov.start` call has to occur before any of the application code is required, so should be at the very top of `spec_helper.rb`